components/common/src/command/package/install.rs

```diff
@@ -191,16 +190,6 @@ where
     P1: AsRef<Path>,
     P2: AsRef<Path>,
 {
-    if env::var_os("HAB_NON_ROOT").is_none() && !am_i_root() {
```

Comment:
Yeah, next time we could've made this a feature flagged behavior behind something like `HAB_FEAT_NON_ROOT` to reflect the fact that it wasn't intended on being permanent.